pkgs/development/python-modules/zephyr-test-management/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  requests,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "zephyr-test-management";
+  version = "0.2.1";
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```
https://github.com/NixOS/nixpkgs/issues/253154

---

Done.